README.md

```diff
@@ -115,6 +115,13 @@ understand what actions will and will not be tolerated.
 Read our [contributing guide](CONTRIBUTING.md) to learn about our development
 process and how to propose bug fixes and improvements.
 
+### Internals
+
+Read our
+[internals](https://memgraph.notion.site/Memgraph-Internals-12b69132d67a417898972927d6870bd2)
+pages to learn more about architecture, how to compile, how to start
+contributing, and find many other guides.
+
```

Comment:
I maybe am a little too pedantic here but why not rephrase it a little bit?

```
Read our [internal](https://memgraph.notion.site/Memgraph-Internals-12b69132d67a417898972927d6870bd2)
docs to learn more about Memgraph's architecture, how to build the project from source and how to start
contributing. All information related to the database, can be found in the aforementioned docs.
```

Reply:
Your version is better! Updated :smile: